swagger.yml

```diff
@@ -1501,6 +1500,143 @@ paths:
         default:
           $ref: '#/responses/error'
   # end site/split_testing/{split_test_id/unpublish}
+  # begin /dns_zones/
+  /dns_zones:
+    post:
+      operationId: createDnsZone
+      consumes:
+        - application/json
+      parameters:
+        - name: DnsZoneParams
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/dnsZoneSetup'
+      responses:
+        '201':
+          description: Created
+          schema:
+            $ref: '#/definitions/dnsZone'
+        default:
+          $ref: '#/responses/error'
+    get:
+      operationId: getDnsZones
+      responses:
+        '200':
+          description: 'get all DNS zones your user account has access to'
+          schema:
+            $ref: '#/definitions/dnsZones'
+        default:
+          $ref: '#/responses/error'
+  # end /dns_zones
+  # begin dns_zones/{zone_id or name}
+  /dns_zones/{zone_id}:
+    parameters:
+      - name: zone_id
+        type: string
+        in: path
+        required: true
+    get:
+      operationId: getDnsZone
+      responses:
+        '200':
+          description: 'get a single DNS zone'
+          schema:
+            $ref: '#/definitions/dnsZone'
+        default:
+          $ref: '#/responses/error'
+    delete:
+      operationId: deleteDnsZone
+      responses:
+        '204':
+          description: 'delete a single DNS zone'
+        default:
+          $ref: '#/responses/error'
+  # end dns_zones/{zone_id or zone name}
+  # begin dns_zones/{zone_id or name}/transfer
+  /dns_zones/{zone_id}/transfer:
+    parameters:
+      - name: zone_id
+        type: string
+        in: path
+        required: true
+      - name: account_id
+        type: string
+        in: query
+        description: 'the account you want to transfer the dns zone to'
```

Comment:
This is the `account_id` of the Dns Zone.

The endpoint requires 2 more params:
- `transfer_account_id` - the account you want to transfer the dns zone to
- `transfer_user_id` - the user you want to transfer the dns zone to (can be the same as the user calling the endpoint)